content/sources/workflow.adoc

```diff
@@ -24,21 +24,20 @@ image::git-organization.png[Git Organization, 600, align=center, link={imagesdir
 This repository is the developers repository. The main development branch is `develop` branch. Depending the contribution kind (one shot contribution or feature contribution), the developer can use a dedicated `feature` branch if needed.
 The most of development activity will be done in this repository by a lot of contributors :
 
-* `maintainers` (formerly known as `sl3`) : To develop fix. These developments are done in dedicated `fix` branches (more details below).
-* `all development teams` : To do one shot contribution and handle feature branch life cycle.
+* `maintainers` (formerly known `Maintenance FT`) : To develop fix. These developments are done in dedicated `fix` branches (more details below).
+* `all other feature teams` : To do one shot contribution and handle feature branch life cycle.
 
 In this repository, `develop` branch and `feature` branch will be under CI.
 
 === Blessed repository
 
 In this repository you can find `stable` branches and release tags.
-Only some people are able to write on this repository :
+Only some people use to write on this repository :
 
 * `maintainers` : To integrate pull request on stable branches.
 * `release manager` : To perform release operations on stable (supported) branches.
-* `keepers` : Repository keepers are `Project Leader`, `Team Leader` and `Technical Leader`. They are able to pull changes from dev to blessed when the develop is stable enough and they can process releases for non supported versions of products (alpha, beta, RCs, …)
 
-The continuous integration is applied on stables branches (`stable/1.0.x`, `stable/1.1.x`, etc) on blessed repository.
+The continuous integration is applied on stables branches (`stable/*`) on blessed repository.
```

Comment:
There is no blessed repository anymore for meeds repositories, we should update this (and the description of the origin/blessed repo mechanism more generally).

Reply:
this mechanism will change anyway in 6.0 (no more exodev, no more synchro, no more exo-addons)
I will enrich the documentation when this is done.
For now, no need to change the current documentation, because this is true even on Meeds-io (the stable branch will be created once 6.0 is released)

Reply:
ok